app/src/shared/errors/InvalidHexStringError.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+export default class InvalidHexStringError extends Error {
```

Comment:
@juhah this is the new error type for the invalid hex string. We throw it in `getValidId` and process (eventually) as `ResourceNotFoundError`.